src/main/kotlin/mate/academy/HelloWorldAnonymousClasses.kt

```diff
@@ -8,6 +8,53 @@ class HelloWorldAnonymousClasses {
     }
 
     fun sayHello(names: List<String>): List<String> {
-        // TODO: implement
+        val greetings = mutableListOf<String>()
+        val englishGreeting = object : HelloWorldGreeting {
+            override fun greet(): String {
+                return "Hello world"
+            }
+
+            override fun greetSomeone(someone: String): String {
+                return "Hello $someone"
+            }
+        }
+
+        val frenchGreeting = object : HelloWorldGreeting {
+            override fun greet(): String {
+                return "Salut tout le monde"
+            }
+
+            override fun greetSomeone(someone: String): String {
+                return "Salut $someone"
+            }
+        }
+
+        val spanishGreeting =  object : HelloWorldGreeting {
+            override fun greet(): String {
+                return "Hola, mundo"
+            }
+
+            override fun greetSomeone(someone: String): String {
+                return "Hola, $someone"
+            }
+        }
+
+        greetings.add(englishGreeting.greet())
```

Comment:
Try to find more expandable solution.
for example, consider case if there are 100 greetings